sympy/stats/rv.py

```diff
@@ -765,6 +765,42 @@ def cdf(expr, condition=None, evaluate=True, **kwargs):
         return result
 
 
+def characteristic_function(expr, condition=None, evaluate=True, **kwargs):
+    """
+    Characterisfic Function of a random expression.
+
+    optionally given a second condition
```

Comment:
This looks like a part of the previous sentence, it doesn't begin with an uppercase letter. Perhaps it should be moved up as a continuation of that sentence. (These docstrings go on display at docs.sympy.org where people don't expect uncapitalized sentence fragments).

Reply:
The same problem occurs in the cdf function above where I copied the docstring from, I'll fix my own but there should be a separate pull request for the other.